src/ui/FacetSlider/FacetSlider.ts

```diff
@@ -1000,9 +1000,8 @@ export class FacetSlider extends Component {
 
   private isFacetEmpty(groupByResults: IGroupByResult, data: IQuerySuccessEventArgs) {
     return (
-      groupByResults == null ||
-      groupByResults.values[0] == null ||
-      groupByResults.values[0].numberOfResults == 0 ||
+      Utils.isNullOrUndefined(groupByResults) ||
+      every(groupByResults.values, value => Utils.isNullOrUndefined(value) || value.numberOfResults === 0) ||
```

Comment:
Is the first half of the condition `Utils.isNullOrUndefined(value)` needed?

Said differently, can a value in the array be `undefined`? Or was the original condition there because we were accessing the first element we wanted to guard against the empty array case?

~Also, what does `every` return in the case where `groupByResults.values` is an empty array?~
Nvm, I saw the unit test for that,